rackcity/utils/change_planner_utils.py

```diff
@@ -21,8 +21,34 @@ def create_asset_cp(sender, **kwargs):
     for related_asset in related_assets_cp:
         related_asset.is_conflict = True
         related_asset.save()
+    # hostname conflicts with hostnames on assetcps
+    instance.hostname_conflict.clear()
```

Comment:
PS what does this line do?

Reply:
so i learned about django related objects where you can access all objects of which the asset is a foriegn key of. this was news to me, not sure if you already knew that. but anyways, this gets all assets in which this current `instance` is a hostname conflict of and clears it, since it might not be a conflict anymore because it is being modified

Reply:
cool